pkg/kstatus/polling/testutil/fake_poller.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 The Kubernetes Authors.
```

Comment:
This is unnecessary--please remove this file.

Reply:
done